x5.md

```diff
@@ -8,7 +8,118 @@ created: 2019-04-18
 updated:
 ---
 
-# Introduction
+# Trust Establishment Protocol Specification Draft
+## Inspiration
+- https://datprotocol.github.io/how-dat-works/
+- 
+
+:::info
+**Trust establishment deals with users verifying they are communicating with who they think they are.**
+:::
+
+## Terms Glossary
```

Comment:
This is good, we should also probably make sure these glossaries are roughly identical across specs. Minor detail tho

Reply:
agreed, see comment about unified variables in other PR